iroha/src/asset.rs

```diff
@@ -45,6 +47,18 @@ impl Asset {
             quantity,
             big_quantity: 0,
```

Comment:
why do we need both quantity and big_quantity?

Reply:
@takemiyamakoto asked about separation of this two values, but I don't think that we will save any memory by this - good question, made an issue for this: https://soramitsu.atlassian.net/browse/RH2-102